Remove t3 channel in IT tests and add it to separate IT class.

sankarpn opened the pull request:
- This PR removes the t3 channel exposed in all IT classes due to intermittent failures caused by port conflict.
- Instead a separate test to expose t3 channel is added to the IT class ItT3Channel.java

Full test suite results
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/3076/testReport/oracle.weblogic.kubernetes/

xiancao reviewed (commented):
Both Jenkins runs failed at oracle.weblogic.kubernetes.ItExternalRmiTunneling.testExternalRmiAccessThruHttpsTunneling. Is this expected?

vanajamukkara commented on integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java:
```diff
@@ -289,6 +326,197 @@ public void testDedicatedModeNoCrd() {
     }
   }
 
+  /**
```
Is it possible to add t3channel to the existing MII domain used in the dedicated mode tests and use that to deploy the application using WLST instead of adding new domain? It saves time by not bringing up a new domain.

sankarpn replied:
I don't want to modify the common model file used for this image MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG because it is shared by many tests. 
Since anyway this test is running alone, the time it take to run this test shouldn't matter IMO.

vanajamukkara commented on integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java:
```diff
@@ -397,6 +625,45 @@ private void verifyDomainNotRunning(String domainNamespace) {
     }
   }
 
+  private static void verifyMemberHealth(String adminServerPodName, List<String> managedServerNames,
```
move this into an util class

sankarpn replied:
This method uses a specific app deployed in the test method. Its not a member for util class.

vanajamukkara commented on integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java:
```diff
@@ -289,6 +326,197 @@ public void testDedicatedModeNoCrd() {
+    int t3ChannelNodePort = assertDoesNotThrow(()
+        -> getServiceNodePort(domain1Namespace, getExternalServicePodName(adminServerPodName), "t3channel"),
+        "Getting admin server t3channel node port failed");
+    logger.info("techannel channel node port: {0}", t3ChannelNodePort);
```
```suggestion
    logger.info("t3channel channel node port: {0}", t3ChannelNodePort);
```

sankarpn replied:
fixed

sankarpn commented:
> 
> 
> Both Jenkins runs failed at oracle.weblogic.kubernetes.ItExternalRmiTunneling.testExternalRmiAccessThruHttpsTunneling. Is this expected?

Thanks for pointing out. Fixed it. The ItExternalRmiTunneling was using a wrong api to get the service port and the test was failing intermittently. Updated the test to use correct api.

vanajamukkara reviewed (approved):
LG.

anpanigr reviewed (commented):
Need to modify the kindtest.sh to exclude ItT3Channel test in Parallel run and include it in sequential run.

anpanigr reviewed (approved):
LGTM
